.travis.yml

```diff
@@ -1,14 +1,11 @@
 sudo: false
 language: ruby
 rvm:
-  - 2.0.0-p648
-  - 2.1.10
-  - 2.2.9
   - 2.3.6
   - 2.4.3
   - 2.5.0
```

Comment:
Do we want to remove 2.5.0 and just test against 2.5.3?

Reply:
Sure